src/index.js

```diff
@@ -126,20 +126,16 @@ class Renderer {
       // { [string]: { error: Error?, html: string, job: Job } }
       // eslint-disable-next-line arrow-body-style
       return axios.post(this.url, item.jobsHash, this.config).then((res) => {
-        return {
-          error: null,
-          results: reduce(res.data, {}, (obj, key) => {
-            const body = res.data[key];
-
-            obj[key] = { // eslint-disable-line no-param-reassign
-              error: body.error,
-              html: body.error ? renderHTML(key, data[key]) : body.html,
-              job: item.jobsHash[key],
-            };
-
-            return obj;
-          }, {}),
-        };
+        const results = res.data.results;
+
+        Object.keys(results).forEach((key) => {
```

Comment:
aren't we using `airbnb-js-shims`, such that we could use `Object.entries` here? not a huge deal either way.